<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>jalmp - just a little music player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>980</width>
      <height>650</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="homePage">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchPage">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>53</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>search</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="playlistsPage">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Playlists</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="artistsPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Artists</string>
      </property>
     </widget>
    </widget>
    <widget class="SingleArtistPage" name="singleArtistPage">
     <widget class="QLabel" name="artistCoverLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="artistNameLabel">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>671</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAlbumButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>View Albums</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="tracklistArea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>911</width>
        <height>351</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tracklist">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>909</width>
         <height>349</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="Line" name="verticalLine">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>3</width>
      <height>650</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="horizontalLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="MediaSlider" name="mediaSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>670</y>
      <width>1000</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaSlider</class>
   <extends>QSlider</extends>
   <header>MediaSlider.hpp</header>
  </customwidget>
  <customwidget>
   <class>SidebarTitle</class>
   <extends>QLabel</extends>
   <header>SidebarTitle.hpp</header>
  </customwidget>
  <customwidget>
   <class>SingleArtistPage</class>
   <extends>QWidget</extends>
   <header>SingleArtistPage.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
